### ISSUE TITLE
Patches list returns wrong SUPEE numbers

### ISSUE THREAD

**Issue by peterjaap** [Hypernode/tools.hypernode.com#3]
For example, the response for 1.7.0.2 (https://tools.hypernode.com/patches/community/1.7.0.2) is;

```json
{
"required": [
"APPSEC-212",
"SUPEE-1533",
"SUPEE-5344",
"SUPEE-5994",
"SUPEE-6285",
"SUPEE-6482",
"SUPEE-6788",
"SUPEE-7405",
"SUPEE-7405 v1.1"
]
}
```

You'll see APPSEC-212 here. That is not a SUPEE patch, because there are different SUPEE patches for different versions for this appsec. For 1.7.0.2 for example, the SUPEE number is SUPEE-2677. So SUPEE-2677 is in the `app/etc/applied.patches.list`, not APPSEC-212.

The same thing goes for SUPEE-7405. The v1.1 patch entry in applied.patches.list doesn't differentiate from the v1.0 version so there is no way knowing of the v1.1 is installed or the v1.0. I suspect this will be the case for SUPEE-8788 v1 and v2 as well. Also, SUPEE-1533 is included in SUPEE-8788 v2 so when SUPEE-8788 v2 is installed, SUPEE-1533 should automatically be marked as installed as well. 

I don't have implementation ideas yet where to catch this; on tools.hypernode.com would be the best bet probably. Or maybe get John to update/change his sheet?

**Comment by gwillem** [comment]
I've changed the behaviour to only return the latest patch version per SUPEE number. 

As for APPSEC, I've filtered them from the spreadsheet as we can't derive information (in an automated fashion) right now.

**Comment by peterjaap** [comment]
Nice work! Any reason why this couldn't be closed?

**Comment by gwillem** [comment]
Was awaiting your comments, haha. For anyone else interested, we discussed that it is semantically more correct to show all the patches, and leave it up to the magerun plugin to decide which patch is not required (eg 8788 v1 should not be required if v2 is installed)

**Comment by gwillem** [comment]
TL;DR: So `tools.hypernode.com` now returns ALL patch revisions again.
